src/FormConfig/EmailConfig.php

```diff
@@ -22,8 +22,8 @@ public function __construct(RenderedSubmission $submission)
         $this->endpoint = $submission->getEndpoint();
         $message = \json_decode($submission->getMessage(), true);
 
-        $this->from = $message['from'];
-        $this->subject = $message['subject'];
+        $this->from = $message['from'] ?? 'noreply@elasticms.eu';
```

Comment:
Do we want an elasticms.eu as default address for everybody in the world using EMS?